db/version_set.cc

```diff
@@ -4889,8 +4889,10 @@ Status VersionSet::DumpManifest(Options& options, std::string& dscname,
               printf("--------------- Column family \"%s\"  (ID %" PRIu32
                      ") --------------\n",
                      cfd->GetName().c_str(), cfd->GetID());
-              printf("%s at level %d\n", files[i]->DebugString(hex).c_str(),
-                     level);
+              auto debugged_string = files[i]->DebugString(hex);
```

Comment:
Revert?

Reply:
Strange, it seems to be generated by I rebase 6.4.tikv, I'll fix it